### PR TITLE
Teddy

### PR DESCRIPTION
**Pull request by gdh8230**
## Summary by CodeRabbit

- **New Features**
	- Real-time status data fetching added to the ArbitrumStatus component.
	- CustomAccordion updated to support JSX.Element for titles and a loading state.
	- TabsManager restructured for improved tab management based on scroll position.

- **Bug Fixes**
	- Improved state management and loading indicators across components.

- **Documentation**
	- Updated interface names for clarity and consistency.
	- Added new dependencies and updated existing ones in the project configuration.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "recoil": "^0.7.7",
     "rxjs": "^7.8.1",
     "styled-components": "^6.1.8",
-    "viem": "^2.21.16"
+    "viem": "^2.21.16",
+    "yarn": "^1.22.22"
```
[the remark this answers is not in the document]
Comment (skskgus) [reply package.json:34]: yarn 왜까셨나요

**Review by skskgus — APPROVED** [review]

File: src/components/arbComponents/arbitrumstatus/ArbitrumStatus.tsx [hunk 2]
```diff
@@ -21,96 +24,142 @@ const getStatusColor = (status: ComponentStatus) => {
...
-              value={component.status}
-              description={component.description}
-              status={component.status}
-            />
-          </Grid>
-        ))}
+        {loading
+          ? // 데이터 로드 중일 때 Skeleton 표시 (고유 key 생성)
+            Array.from(new Array(5)).map(() => (
+              <Grid item key={crypto.randomUUID()} xs={2}>
```
[the remark this answers is not in the document]
Comment (gdh8230) [reply src/components/arbComponents/arbitrumstatus/ArbitrumStatus.tsx:90]: 이렇게하면 타입오류남

[the remark this answers is not in the document]
Comment (gdh8230) [reply src/components/arbComponents/arbitrumstatus/ArbitrumStatus.tsx:90]: [{
	"resource": "/Users/gong/Downloads/ivp-front/src/components/arbComponents/arbitrumstatus/ArbitrumStatus.tsx",
	"owner": "eslint",
	"code": {
		"value": "react/no-array-index-key",
		"target": {
			"$mid": 1,
			"external": "https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-array-index-key.md",
			"path": "/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-array-index-key.md",
			"scheme": "https",
			"authority": "github.com"
		}
	},
	"severity": 8,
	"message": "Do not use Array index in keys",
	"source": "eslint",
	"startLineNumber": 90,
	"startColumn": 31,
	"endLineNumber": 90,
	"endColumn": 36
}]

**Review by gloomydumber — APPROVED** [review]
LGTM

**Pull request merged**